serverless/event_workflows/serverless-create-kn-trigger.adoc

```diff
@@ -0,0 +1,27 @@
+[id="serverless-create-kn-trigger_{context}"]
+= Creating a trigger using `kn`
+
+All events that are sent to a broker will be sent to all of the subscribers of that broker by default.
```

Comment:
Could we maybe clarify this sentence a bit? while technically true (the best kind of true), it gave me the impression we were trying to say you use subscriptions (the channel related resource type) to specify sinks being attached to brokers and they're all enabled by default.  I think we're trying to say is: unless a filter is specified on the trigger spec, the trigger will forward all events from the broker to that sink, right?

Perhaps (and feel free to bikeshed): "By default, a Trigger will forward all events sent to a broker to it's specified subscribers." ?

Reply:
Makes sense to me, thanks @lberk ! I've made some modifications in line with this so lmk if that looks better to you.